app/components/edit-claim-conditions/EditClaimConditions.tsx

```
@@ -0,0 +1,238 @@
+        client,
+        chain: EditonDropContractDeployedChain,
+        transactionHash,
+      });
+
+      return receipt;
+    } catch (err) {
+      if (err instanceof Error) {
+        throw new Error(err.message);
+      } else {
+        throw err;
+      }
```

Biome (1.9.4)

[error] This else clause can be omitted because previous branches break early.

Unsafe fix: Omit the else clause.


(lint/style/noUselessElse)